core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -57,7 +57,7 @@ def convert_endpoint(endpoint: Endpoint) -> str:
         ]:
             raise ValueError(f"Master endpoint of type {master.connection_type} not supported on MavlinkRouter.")
 
-        log = f"--log {self.logdir().resolve()}"
+        log = f"--log {self.logdir().resolve()} --tlog {self.logdir().resolve()}"
```

Comment:
this should be just `log = f"--tlog {self.logdir().resolve()}"` right?

Reply:
I was aiming to have both logs, but Ok.

Reply:
I'm not against it. We just need to keep this in sync with `mavproxy`.